Documentation/getting-started.md

```diff
@@ -1,71 +1,97 @@
 
-## Add the AEP Consent extension to your app
+## Add Consent to your app
 
-### Download and import the Consent extension
+The Consent for Edge Network extension depends on the following extensions:
+* [Mobile Core](https://github.com/adobe/aepsdk-core-android)
+* [Edge Network](https://github.com/adobe/aepsdk-edge-android) (required for handling request to Adobe Edge Network, including consent preferences updates)
+* [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) (peer dependency for the Edge Network extension)
 
-1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file:
 
-  ```java
+```java
 implementation 'com.adobe.marketing.mobile:core:2.+'
 implementation 'com.adobe.marketing.mobile:edge:2.+'
 implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
 implementation 'com.adobe.marketing.mobile:edgeconsent:2.+'
-  ```
+```
+
+> **Warning**
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/GradleDependencies.md) for managing gradle dependencies.
 
-2. Import the Mobile Core and Edge extensions in your Application class.
+2. Import the libraries:
 
-  ```java
+#### Java
+```java
 import com.adobe.marketing.mobile.MobileCore;
 import com.adobe.marketing.mobile.Edge;
 import com.adobe.marketing.mobile.edge.identity.Identity;
 import com.adobe.marketing.mobile.edge.consent.Consent;
-import com.adobe.marketing.mobile.LoggingMode;
-import android.app.Application;
-import java.util.Arrays;
-import android.util.Log;
-  ```
+```
+
+#### Kotlin
 
-3. Register Edge extensions with Mobile Core
+```kotlin
+import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.Edge
+import com.adobe.marketing.mobile.edge.identity.Identity
+import com.adobe.marketing.mobile.edge.consent.Consent
+```
 
-  ```java
-public class MobileApp extends Application {
+## Register Edge extensions with Mobile Core
 
-  private final String ENVIRONMENT_FILE_ID = "";
+```java
+public class MainApp extends Application {
 
-@Override
+  private final String ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID";
+
+	@Override
 	public void onCreate() {
 		super.onCreate();
-		MobileCore.setApplication(this);
 
-		MobileCore.setLogLevel(LoggingMode.VERBOSE);
+		MobileCore.setApplication(this);
 		MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
 
-		// register Adobe extensions
 		MobileCore.registerExtensions(
-			Arrays.asList(Consent.EXTENSION, Edge.EXTENSION, Assurance.EXTENSION),
-			o -> Log.d("MobileApp", "Mobile SDK was initialized")
+			Arrays.asList(Consent.EXTENSION, Identity.EXTENSION, Edge.EXTENSION),
+			o -> Log.d("MainApp", "AEP Mobile SDK was initialized")
```

Comment:
Avoiding usage of the AEP acronym, change to "Adobe Experience Platform Mobile SDK was initialized"?
 Same for Kotlin.